addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -22,6 +22,10 @@ const LinkPopoverWidget = Widget.extend({
         this.$target = $(target);
         this.href = this.$target.attr('href'); // for template
         this._dp = new DropPrevious();
+
+        this.$target.on('dblclick', (ev) => {
+            $('#toolbar #create-link').click();
```

Comment:
+ adapt test...

There are an explicit test: "Ensure nothing happened on double click (except showing popover)"

Probably to remove ;)  and ask to @rdeodoo, if he did smthing special to remove the behavior 'on double click' before to 'reimplement' it

Reply:
Reintroducing double click to edit link shouldn't be done in `link_popover_widget` which is only in charge of showing the popover, and shouldn't be handling editor events.
It should be a behavior of the web editor itself as it was before I removed it, see [here](https://github.com/odoo/odoo/pull/64756/files#diff-43f0ca2387d343389773d1f8fa0a22a678fc31afc6b93405fb3f8009418d379aL890).
Note that the `create_dblclick_feature` method was also in charge of creating the tooltip, which should also be restored.

But why are we reintroducing it when @pierre-pde and SBU asked to remove that behavior a few months ago?
https://www.odoo.com/web#cids=1&id=2439860&menu_id=&model=project.task&view_type=form
(see point 3.)
![pde](https://user-images.githubusercontent.com/30048408/131341747-864d4a09-2b1c-40c8-8f39-5917b1f7be88.png)

Reply:
@rdeodoo, it was removed because at that time, double click was opening a new tab.
Now it's displaying options at the bottom of style tab. 
Though I wonder if we shouln't display those link options every time a link is clicked on... (single click vs double click).

Reply:
> Though I wonder if we shouln't display those link options every time a link is clicked on... (single click vs double click).

Sounds ok to me, I don't see an issue doing that.
(and technically it would mean not restoring a double click event but hooking that where we are starting the popover)